test/k8sT/Services.go

```diff
@@ -301,14 +339,18 @@ var _ = Describe("K8sServicesTest", func() {
 			// teardown if any step fails.
 			_ = kubectl.Delete(demoYAML)
 			_ = kubectl.Delete(echoSVCYAML)
+			if dualStackSupportEnabled {
```

Comment:
This looks odd, we can't "know" the order the variable will be initialized so it would be better executing `helpers.DualStackSupported()` directly here.